data/desktop.metainfo.xml.in

```diff
@@ -43,6 +43,16 @@
   <update_contact>contact_at_elementary.io</update_contact>
 
   <releases>
+    <release version="8.2.0" date="2024-12-02" urgency="medium">
```

Comment:
I believe you should also update the application version in the Meson configuration.

Reply:
Usually we don't update that until we actually make a new release. I'm just adding a new release tag here to get translations